Lib/glyphsLib/cli.py

```diff
@@ -0,0 +1,226 @@
+        ),
+    )
+    group.add_argument(
+        "--no-normalize-ufos",
+        action="store_false",
+        help=(
+            "Skip normalizing UFOs with ufonormalizer, which would avoid "
+            "differences due to spacing, reordering of keys, etc."
+        ),
+    )
+
+    parser_ufo2glyphs = subparsers.add_parser(
+        "ufo2glyphs",
+        help="Convert one designspace file or one or more UFOs to a Glyphs file.",
+    )
+    parser_ufo2glyphs.set_defaults(func=ufo2glyphs)
+    parser_ufo2glyphs.add_argument(
+        "--version", action="version", version="glyphsLib %s" % (glyphsLib.__version__)
+    )
+    parser_ufo2glyphs.add_argument(
+        "designspace_file_or_UFOs",
+        nargs="+",
+        metavar="DESIGNSPACE_FILE_OR_UFOS",
+        help=(
+            "A single designspace file *or* one or more UFOs to convert to "
+            "a Glyphs file."
+        ),
+    )
+    parser_ufo2glyphs.add_argument(
+        "--output-path", help="The path to write the Glyphs file to."
+    )
+    group = parser_ufo2glyphs.add_argument_group(
+        "Roundtripping between UFOs and Glyphs"
+    )
+    group.add_argument(
+        "--no-preserve-glyphsapp-metadata",
+        action="store_false",
+        help=(
+            "Skip preserving Glyphs metadata in master UFOs and designspace "
+            "file, which would be used to minimize differences when "
+            "roundtripping between Glyphs and UFOs."
+        ),
+    )
+    group.add_argument(
+        "--enable-last-change",
+        action="store_false",
+        help="Store modification timestamp in glyphs. Unnecessary when using Git.",
+    )
+    group.add_argument(
+        "--enable-automatic-alignment",
+        action="store_false",
+        help="Enable automatic alignment of components in glyphs.",
+    )
+
+    options = parser.parse_args(args)
+
+    if "func" in vars(options):
+        return options.func(options)
+    else:
+        parser.print_help()
+
+
+def glyphs2ufo(options):
+    """Converts a Glyphs.app source file into UFO masters or UFO instances and
+    MutatorMath designspace."""
+    if options.output_dir is None:
+        options.output_dir = os.path.dirname(options.glyphs_file)
+
+    if options.instance_dir is None:
+        options.instance_dir = os.path.join(options.output_dir, "instance_ufos")
+
+    if options.designspace_path is None:
+        options.designspace_path = os.path.join(
+            options.output_dir,
+            os.path.basename(os.path.splitext(options.glyphs_file)[0]) + ".designspace",
+        )
+
+    glyphsLib.build_masters(
+        options.glyphs_file,
+        options.output_dir,
+        options.instance_dir,
+        designspace_path=options.designspace_path,
+        minimize_glyphs_diffs=options.no_preserve_glyphsapp_metadata,
+        propagate_anchors=options.propagate_anchors,
+        normalize_ufos=options.no_normalize_ufos,
+    )
+
+    return 0
```

Comment:
this is not required, since `sys.exit(None)` is the same as `sys.exit(0)`, but doesn't do any harm